### PR TITLE
Adding project explanation

### PR DESCRIPTION
**Pull request by thegreystone**
...as suggested in the Twitter discussion. :)

Progress
--------
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

Approvers
---------
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**)

### CODE REVIEW

**Comment by edvbld** [comment]
:tada: :rofl:

**Comment by thegreystone** [comment]
That's a great band name right there - Indecisive Bots!

**Comment by thegreystone** [comment]
Here is the twitter discussion:
https://twitter.com/emilianbold/status/1143175227616239618?s=21

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # OpenJDK Project Skara
 
+The goal of Project Skara is to investigate alternative SCM and code review
+options for the JDK source code, including options based upon Git rather than
```
Comment (edvbld) [inline README.md]:
```suggestion
options for the OpenJDK source code, including options based upon Git rather than
```

**Comment by edvbld** [comment]
Thanks Marcus, the change looks good, but if you want to add me as contributor you need to use the `/contributor` command in a comment to list me as co-author:

```
/contributor add Erik Duveblad <ehelin@openjdk.java.net>
```

**Comment by edvbld** [comment]
Heh, I now realized (thanks to Robin) that you just incorporated my suggestion and that GitHub then automatically added me as co-author :smiley: _If_ you want to add me as co-author then you have to do as I described above, but I'm fine if you want to be sole author :smile_cat:

**Comment by thegreystone** [comment]
Done! :)

**Comment by edvbld** [comment]
@thegreystone as the `openjdk[bot]` described above, you need to type the `/integrate` command in a comment to highlight that this PR is ready to be sponsored :wink:

**Comment by thegreystone** [comment]
Oh dear.

**Pull request closed without merging**